Tasks/AzureRmWebAppDeployment/Utility.ps1

```diff
@@ -126,6 +127,10 @@ function Get-MsDeployCmdArgs
         $msDeployCmdArgs += [String]::Format(' -skip:Directory="\\App_Data"')
     }
 
+    if( -not [String]::IsNullOrEmpty($setParametersFile)){
```

Comment:
I actually do this check in the main file (properly now) - by the time this script is called, the path will be empty if the user didn't specify a SetParamsFile.